docs/fundamentals/index.yml

```diff
@@ -97,16 +103,6 @@ landingContent:
           - text: App publishing
             url: ../core/deploying/index.md
 
-  # Card
-  - title: Migrate from .NET Framework
```

Comment:
What's the idea behind combining this section with the overview section? @adegeo Are you okay with this?

Reply:
We are limited to only 12 cards on this landing page. My idea was to combine porting, migration, and upgrade assistant links into Getting started with .NET Core - as they're really the same. If you're just getting started with .NET Core you might be coming from .NET framework. Also, none of the Runtime libraries (extensions and fundamentals) were linked in the landing page - which is a huge miss.

Reply:
As long as it's still linked to from somewhere, I'm fine with it.